hbase_sql/src/test/java/com/splicemachine/mrio/api/core/SplitRegionScannerIT.java

```diff
@@ -171,7 +168,7 @@ public void simpleScan() throws SQLException, IOException, InterruptedException
         Assert.assertEquals("Did not return all rows ",ITERATIONS,i);
 	}
 
-    @Test
+    @Test @Ignore("DB-7320")
```

Comment:
No need to change it but you could have ignored the whole file by adding the annotation to the class definition.